### DISCUSSION TITLE
For go around there was no full throttle since Dev cdf7d1d

### DISCUSSION THREAD

**Discussion by Adi-18** [flybywiresim/aircraft#7034] [Help]
I fly manual, only with A-THR for training touch-and-go

To day I have to go around and press throttle to maximum, but the speed keeps to ~130 kt.
I could not pres to unmanaged speed for manual setting and no reaction when I pull throttle forward.

Now before stall I was possible to keep the level an land 😅

What could it be?

**Comment by tracernz** [comment]
For a go-around you need to advance thrust to TOGA to engage go-around phase and SRS mode... Speed is up to you to follow the FD pitch orders in SRS mode, as the thrust is fixed.

**Comment by Adi-18** [comment]
Thank you for the answer. The point is, I press throttle to maximum that means to TOGA.
But there was no reaction from A320

**Comment by Adi-18** [comment]
I changed the title. Today speed stays on ~280 kt for final landing. 
Its nothing to do with go around.

This time it was not possible to land because of overspedd 😒

I never hat this problem before

**Comment by Adi-18** [comment]
I fly manual, only using A-THR and LS for training touch-and-go

To day I'll to training touch-and-go. After touch I press throttle to maximum for TOGA. after climb, i start to prepare for final landing. after a while, the speed began to sink. Pressing to unmanaged speed for manual setting and pull throttle forward has no effects.

What could it be wrong?

Please see  the video:  https://youtu.be/zJ8DmmV7nUY

**Comment by ExampleWasTaken** [comment]
Well I guess it's realistic that the engines turn off when there's no fuel left :/

**Reply by Adi-18** [reply]
> Well I guess it's realistic that the engines turn off when there's no fuel left :/

Yes it make sense. But shouldn't there raise an alarm in that case, when its on minimum?
